.github/workflows/ci.yaml

```diff
@@ -0,0 +1,12 @@
+name: CI
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test-actions/setup-nextstrain-cli:
+    name: test: actions/setup-nextstrain-cli
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ./actions/setup-nextstrain-cli
```

Comment:
Throw a `run: nextstrain --version` step in here?

Reply:
Oh I guess it is already in the called workflow. But maybe good to check that at least `nextstrain` is available in this context.

Reply:
Ah, the action already does that.

Reply:
@victorlin Added `run: nextstrain version --verbose` after the action in https://github.com/nextstrain/.github/commit/226a9812f6944e1120b5d08149158cb0cca8252b. ([example output](https://github.com/nextstrain/.github/runs/6202728220?check_suite_focus=true)).